app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -95,79 +109,93 @@ function initializePodcastPlayback() {
           stopRotatingActivePodcastIfExist();
           loadAndPlayNewPodcast(episodeSlug);
         }
-      }
-    })
+      };
+    });
   }
 
   function changePlaybackRate(audio) {
     var el = getById('speed');
     var speed = parseFloat(el.getAttribute('data-speed'));
     if (speed == 2) {
       el.setAttribute('data-speed', 0.5);
-      el.innerHTML = '0.5x'
-      audio.playbackRate = 0.5
+      el.innerHTML = '0.5x';
+      audio.playbackRate = 0.5;
     } else {
-      el.setAttribute('data-speed', speed+0.5);
-      el.innerHTML = speed+0.5+"x"
-      audio.playbackRate = speed+0.5
+      el.setAttribute('data-speed', speed + 0.5);
+      el.innerHTML = speed + 0.5 + 'x';
+      audio.playbackRate = speed + 0.5;
     }
   }
 
   function changeStatusMessage(message) {
-    if(getById(`status-message-${window.activeEpisode}`)) {
+    if (getById(`status-message-${window.activeEpisode}`)) {
       getById(`status-message-${window.activeEpisode}`).innerHTML = message;
-    } else if (message === 'loading' && document.querySelector('.status-message')) {
+    } else if (
+      message === 'loading' &&
+      document.querySelector('.status-message')
+    ) {
       document.querySelector('.status-message').innerHTML = message;
-    } else {
-
     }
   }
 
   function applyOnbeforeUnloadWarning() {
-    var message = 'You are currently playing a podcast. Are you sure you want to leave?'
+    var message =
+      'You are currently playing a podcast. Are you sure you want to leave?';
     window.onclick = function(event) {
-      if (event.target.tagName === 'A' && !event.target.href.includes('https://dev.to')) {
+      if (
+        event.target.tagName === 'A' &&
+        !event.target.href.includes('https://dev.to') &&
+        !event.ctrlKey
```

Comment:
Actually, this is the only change I made. Other are auto-fixes by `eslint`.

Reply:
Could you add `&& !event.metaKey` for Mac keyboards? Happy to pull this down to check if you don't have a Mac to test the functionality.